src/App.js

```diff
@@ -17,45 +20,117 @@ What should a cart item look like? 🤔
 console.log(initialStoreItems)
 
 export default function App() {
+ 
+  const [cart, setCart] = useState([])
   // Setup state here...
 
+  const addToCart = selection => {
```

Comment:
Wow, a super lean way to write a function, I wasn't aware that this was possible. Nice!